provider/resource_service.go

```diff
@@ -144,6 +144,7 @@ func (r *resourceService) Schema(ctx context.Context, req resource.SchemaRequest
 							Description: "User-defined description for the KPI. ",
 							Optional:    true,
 							Computed:    true,
+							Default:     stringdefault.StaticString(""),
```

Comment:
I tried to avoid stringdefaults to "", scraper will dump this as ```description: null```. But it is really not common use case :)

Reply:
I'm not aware of the scrape issue, but it sounds very minor.
Not having the default "" here is preventing description from being removed in some cases (like the one added in the new test case) and also causing some unnecessary drifts and terraform warnings.